templates/vue/src/components/__tests__/NodeModal.spec.js

```diff
@@ -1,63 +1,63 @@
-import { fireEvent } from "@testing-library/vue"

```

Comment:
Was this an auto format thing? I don't see any differences.

Reply:
@narendrasss I a little embarrassed to admit that I have no idea why this is showing as a change. I checked and the content of the files is exactly the same, even white-space. Any tips on helping git out here?